typeDefinition/ElementChain.d.ts

```diff
@@ -95,7 +96,8 @@ interface ElementEvalModifiers<T> extends
 {}
 
 interface ElementBaseQueryChain<TSelf> extends
-	BaseChain<TSelf, ElementQueryResult, ElementAbandonedChain>
+	BaseChain<TSelf, ElementQueryResult, ElementAbandonedChain>,
+	ElementProps
```

Comment:
You should remove it. You need replace `type ElementQueryResult = string;` to `type ElementQueryResult = ElementProps;`